eo-maven-plugin/src/main/resources/org/eolang/maven/phi/to-phi.xsl

```diff
@@ -27,9 +27,7 @@ SOFTWARE.
   <xsl:output encoding="UTF-8" method="text"/>
   <!-- Variables -->
   <xsl:variable name="aliases" select="program/metas/meta/part[last()]"/>
-  <xsl:variable name="number-pattern">
-    <xsl:value-of select="'^[0-9][1-9]*$'"/>
-  </xsl:variable>
+  <xsl:variable name="number-pattern" select="'^[0-9][1-9]*$'"/>
```

Comment:
@maxonfjvipon I believe, `10` won't match this pattern. Is it intended to be so?

Reply:
@yegor256 no, it's a mistake